zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -99,6 +102,13 @@ public Either<RejectionType, Void> isAuthorized(final AuthorizationRequest reque
     }
   }
 
+  private boolean isAuthorizedAnonymousUser(final TypedRecord<?> command) {
```

Comment:
🔧 It'd be great if we add some javadoc here to what an authorized anonymous user is and why we have it. I'm sure this will cause some confusion 😄

Reply:
Sure. I added some javadoc. As mentioned, I am also happy to change the term `anonymously` to something more meaningful if it would help to make it more clear.

Reply:
It's fine, thanks for adding doc 👍 I thought about anonymous and I can't think of a better term. I think it describes what it is, someone who is authorized to do an action yet anonymous for our system.